tests-functional/tests/test_positive_values_user_flow.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
+
+from pages.all_aboard import AllAboard
+from pages.firstrun import Firstrun
+from pages.sidebar import Sidebar
+
+
+@pytest.fixture
+def complete_firstrun(base_url, selenium):
+    """ Completes /firstrun interactions to start on-boarding"""
+    page = Firstrun(selenium, base_url).open()
+    page.complete_firstrun_values_flow()
+    return page
```

Comment:
I see you're not using the return value. Unless you plan to, you could make this an autouse fixture, which means it will be executed for all tests in this file and does not need to be mentioned explicitly in the test method signature.